athena-elasticsearch/src/main/java/com/amazonaws/connectors/athena/elasticsearch/ElasticsearchMetadataHandler.java

```diff
@@ -277,6 +353,6 @@ protected Field convertField(String fieldName, String glueType)
     {
         logger.info("convertField - fieldName: {}, glueType: {}", fieldName, glueType);
 
-        return GlueFieldLexer.lex(fieldName, glueType.toLowerCase());
+        return GlueFieldLexer.lex(fieldName, glueType, mapper);
```

Comment:
seems you lost the .toLowerCase() are you doing that somewhere else now? I ask because Presto will not take kindly to mixed case (or incorrect case) and i think that is why we had toLowerCase() here.

Reply:
Not needed here. toLowerCase() is called from within the lex implementation and the mapper implementation.